_posts/homepage/2018-04-12-the-conference.md

```diff
@@ -2,6 +2,7 @@
 layout: post
```

Comment:
Is the date in the filename arbitrary or intentional?

Reply:
The date has to be in there (as jekyll is mostly focussed on blogs), but other than that it doesn't really matter. It might affect the ordering of the stories on the homepage though